build.gradle

```diff
@@ -171,12 +173,14 @@ dependencies {
   functionalTestCompile sourceSets.main.runtimeClasspath
 
   smokeTestCompile sourceSets.main.runtimeClasspath
+  smokeTestCompile sourceSets.test.runtimeClasspath
+  smokeTestCompile group: 'io.rest-assured', name: 'rest-assured', version: '3.0.7'
 }
 
-mainClassName = 'uk.gov.hmcts.reform.demo.Application'
+mainClassName = 'uk.gov.hmcts.reform.bulkscanning.Application'
 
 bootJar {
-  archiveName = 'spring-boot-template.jar'
+  archiveName = 'bulk-scanning.jar'
```

Comment:
should we name the repo as `scanned-doc-processor` ? As actually it is not doing scanning ? May be good if we do it sooner before going into prod.

Reply:
:+1: Let's run this idea past the team and address in another PR.